src/cmd/build.js

```diff
@@ -31,8 +31,9 @@ function defaultPackageCreator(
           filter: (...args) => fileFilter.wantFile(...args),
         })
         .then((buffer) => {
+          let extension = 'zip';
           let packageName = safeFileName(
-            `${manifestData.name}-${manifestData.version}.xpi`);
+            `${manifestData.name}-${manifestData.version}.${extension}`);
```

Comment:
might as well just make this `${manifestData.name}-${manifestData.version}.zip` -- no need for a variable